vendor/github.com/multiformats/go-multihash/multihash.go

```diff
@@ -58,6 +58,8 @@ const (
 	BLAKE2S_MIN = 0xb241
 	BLAKE2S_MAX = 0xb260
 
+	BMT_SHA3 = 0xb270
```

Comment:
Why are we changing a vendored library? If this is an update to the library from upstream, we should update `vendor.json` with the new revision. If this is an update on our side, we should fork the library, or bundle it with swarm in another path but `/vendor`

Reply:
@nonsense It is our change. Which way would you prefer, forking it or bundling it with swarm?

Reply:
`multihash` lib is just a few files, each 200 lines of code. I'd embed it as part of our codebase. Make sure you keep the LICENSE file in the folder.

No need to maintain a fork.

Reply:
we should do this once the bmt hash is a little more mature, till then just do as @nonsense suggests